{
  "artifacts": [],
  "command": "main",
  "experiment": {
    "base_dir": "/home/gradient/nico/baselines/script",
    "dependencies": [
      "baselines==0.1.5",
      "gym==0.10.5",
      "numpy==1.14.3",
      "sacred==0.7.4",
      "tensorflow==1.8.0"
    ],
    "mainfile": "run_rllab_pois_sacred.py",
    "name": "POIS",
    "repositories": [
      {
        "commit": "3fff3d1700e200bdc292c6993e0c4df055ba6ffb",
        "dirty": false,
        "url": "https://github.com/T3p/baselines.git"
      }
    ],
    "sources": [
      [
        "run_rllab_pois_sacred.py",
        "_sources/run_rllab_pois_sacred_7cfd8633032921ff13976e095d51bd3b.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/gradient/anaconda3/envs/baselines/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py\", line 510, in _apply_op_helper\n    preferred_dtype=default_dtype)\n",
    "  File \"/home/gradient/anaconda3/envs/baselines/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 1104, in internal_convert_to_tensor\n    ret = conversion_func(value, dtype=dtype, name=name, as_ref=as_ref)\n",
    "  File \"/home/gradient/anaconda3/envs/baselines/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 947, in _TensorTensorConversionFunction\n    (dtype.name, t.dtype.name, str(t)))\n",
    "ValueError: Tensor conversion requested dtype int32 for Tensor with dtype float32: 'Tensor(\"sub_16:0\", shape=(), dtype=float32)'\n",
    "\nDuring handling of the above exception, another exception occurred:\n\n",
    "Traceback (most recent call last):\n",
    "  File \"/home/gradient/.local/lib/python3.5/site-packages/sacred/config/captured_function.py\", line 46, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"script/run_rllab_pois_sacred.py\", line 149, in main\n    max_iters=max_iters)\n",
    "  File \"script/run_rllab_pois_sacred.py\", line 124, in train\n    clipping=clipping, entcoeff=entcoeff)\n",
    "  File \"/home/gradient/nico/baselines/baselines/pois/pois.py\", line 535, in learn\n    ent_f = tf.minimum(0, 1-tf.abs(tf.reduce_mean(iw) - 1)) * entcoeff\n",
    "  File \"/home/gradient/anaconda3/envs/baselines/lib/python3.5/site-packages/tensorflow/python/ops/gen_math_ops.py\", line 4636, in minimum\n    \"Minimum\", x=x, y=y, name=name)\n",
    "  File \"/home/gradient/anaconda3/envs/baselines/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py\", line 546, in _apply_op_helper\n    inferred_from[input_arg.type_attr]))\n",
    "TypeError: Input 'y' of 'Minimum' Op has type float32 that does not match type int32 of argument 'x'.\n"
  ],
  "heartbeat": "2018-08-29T15:29:26.828253",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) CPU E5-4610 v2 @ 2.30GHz",
    "hostname": "nash",
    "os": [
      "Linux",
      "Linux-4.4.0-130-generic-x86_64-with-debian-stretch-sid"
    ],
    "python_version": "3.5.5"
  },
  "meta": {
    "command": "main",
    "options": {
      "--beat_interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": true,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print_config": false,
      "--priority": null,
      "--queue": false,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "env=inverted-pendulum",
        "iw_method=is",
        "bound=max-d2",
        "policy=linear",
        "logdir=ip_linear",
        "delta=0.9",
        "center=True",
        "gamma=0.99",
        "njobs=1",
        "seed=109",
        "entropy=1.0"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2018-08-29T15:29:19.569873",
  "status": "FAILED",
  "stop_time": "2018-08-29T15:29:26.832821"
}